src/components/Modal/modal.scss

```diff
@@ -55,6 +55,10 @@
     border-bottom-right-radius: 5px;
```

Comment:
There are some values of box-shadow and border that are still in pixels, instead of rem. I don't know if it's necessary to change...